buildenv/jenkins/JenkinsfileBase

```diff
@@ -6,7 +6,10 @@ import org.tap4j.model.TestSet;
 def makeTest(testParam) {
 	String tearDownCmd = "$RESOLVED_MAKE; \$MAKE -f ./aqa-tests/TKG/testEnv.mk testEnvTeardown"
 	// Note: keyword source cannot be used in Jenkins script. Therefore, using "." instead.
```

Comment:
This comment should be moved into the `if` block too

Reply:
To make sure, do you want me to move the import otf.tap4j.model.TestSet comment inside my if statement on line 10 or before it on line 9?

Reply:
This comment `// Note: keyword source cannot be used in Jenkins script. Therefore, using "." instead.` is for `makeTestCmd = "$RESOLVED_MAKE;cd ./aqa-tests; . ./scripts/testenv/testenvSettings.sh;cd ./TKG; \$MAKE $testParam"`, so they should be together.